src/backend/distributed/commands/utility_hook.c

```diff
@@ -897,7 +897,7 @@ ShouldCheckUndistributeCitusLocalTables(void)
 		return false;
 	}
 
-	if (IsCitusInitiatedRemoteBackend())
+	if (IsCitusInternalBackend())
```

Comment:
I think we still use `IsCitusInitiatedRemoteBackend()` in other places and now we also introduce `IsCitusInternalBackend()`. It is probably non-trivial for readers of these functions and their meanings (and also when to use which one). I'd rather eliminate this confusion.

In that regard, I think we can do:
1) [optional] Rename `IsCitusInitiatedRemoteBackend()` to `IsCitusInternalBackend()` -- because the latter is much nicer name
2) Use `|| IsRebalancerInitiatedBackend()` where they are actually needed such all the places we use `IsCitusInternalBackend()` in this current commit

I think with that, the intention of each function use would be explicit.

Reply:
Done.